src/parser/jobs/drk/modules/ResourceSimulator.js

```diff
@@ -24,23 +24,30 @@ const BLOODSPILLER_BLOOD_COST = 50
 // Meters
 // ------
 const MAX_MANA = 9480
+const MANA_REGEN_PERCENT_PER_TICK = 0.02
```

Comment:
Linter is dumb with constants and allows magic on single lines but not multi (like you have below). May want to use `2/100` here to be consistent but it's kinda yolo in this case.